src/lib/registration/storage.rb

```diff
@@ -37,7 +38,7 @@ class RegCodes < Struct.new(:reg_codes)
 
       def initialize
         self.reg_codes = if Stage.initial
-          reg_codes_from_usb_stick || {}
+          reg_codes_from_usb_stick || reg_code_from_install_inf
```

Comment:
This one single line of code that is [not covered by tests](https://coveralls.io/builds/14865538/source?filename=src%2Flib%2Fregistration%2Fstorage.rb#L41)... contains a typo.

Reply:
ok